DittoCombineExample/DittoCombineExample/DittoManager.swift

```diff
@@ -11,9 +11,10 @@ final class DittoManager {
     private var cancellables = Set<AnyCancellable>()
 
     private init() {
-        let identity = DittoIdentity.offlinePlayground(appID: "live.ditto.skyservice")
-        ditto = Ditto(identity: identity)
-        try! ditto.setLicenseToken("")
+        ditto = Ditto()
+        try! ditto.setOfflineOnlyLicenseToken(
+            "o2d1c2VyX2lkdTExODE0NDU2ODIwNjc3NjI1MjAxN2ZleHBpcnl4GDIwMjMtMDEtMTVUMjA6MjM6MDAuMzg0WmlzaWduYXR1cmV4WGt5REdkbkhid3RqWjl1Nk5uTllhM1g1K1ZMc2ZWU1RTOUR5Q1p4TlVYUXhWRzM1RENWWHJBWWd0N3NUTEZVbmtmWEJsMXhoL1JOeWRZZXVZeEV0L1NnPT0="
```

Comment:
Can you remove the offline only license token from here so that developers control their own tokens?

Reply:
Good catch; I'll remove it. Actually, this probably shouldn't be a PR yet since the app does nothing visibly. So far I've just updated to v3 and made it buildable.

Reply:
Updating to V3 is useful though because some people may just copy/paste this and we want them to be on the latest version

Reply:
Resolved in commit f711c3e0b61f31c012e86000b0db66c150abc4a2

Reply:
Is this a 30 day offline license token? Just making sure we aren't leaking a longer lived key here